src/sequence.rs

```diff
@@ -77,6 +77,42 @@ pub fn new_sequence<P: AsRef<Path>>(
         .map(|_| ())
 }
 
+pub fn remove_sequence<P: AsRef<Path>>(admin_key_path: P, name: &str) -> Result<(), Error> {
+    
+    // Check that the admin has sufficient privileges
+    try!(utils::validate_admin(&admin_key_path));
+
+    // Make sure the name is valid (needed since it will be used in a file path)
+    try!(validate_seq_name(name));
+
+    // Make the name of the sequence's directory
+    let mut sequence_dir = String::from("seq_");
+    sequence_dir.push_str(&name);
+    let sequence_dir = sequence_dir;
+
+    // Remove sequence's directory
+    let sequence_dir_path = Path::new(&sequence_dir);
+    if sequence_dir_path.exists() && sequence_dir_path.is_dir() {
+        let _ = fs::remove_dir_all(&sequence_dir_path)
+            .expect("Error removing sequence directory");
+    }
+
+    // Remove sequence from project
+    let project = try!(utils::read_protonfile(None::<P>));
```

Comment:
You should move this section before `remove sequence's directory`, because if something goes wrong in the protonfile, its less damaging than something going wrong in an IO op. Rule of thumb: less risky things first.